Dockerfile

```diff
@@ -0,0 +1,51 @@
+# -------------------
+# The build container
+# -------------------
+FROM debian:bullseye-slim AS build
+
+RUN apt-get update && \
+  apt-get -y --no-install-recommends install \
+    build-essential \
+    clang \
+    cmake \
+    libcurl4-openssl-dev \
+    libfftw3-dev \
+    libusb-1.0-0-dev \
+    pkg-config \
+    unzip && \
+  rm -rf /var/lib/apt/lists/*
+
+ADD https://github.com/steve-m/librtlsdr/archive/master.zip /root/librtlsdr-master.zip
```

Comment:
I'm not sure of the official repository for rtl-sdr, I pick the osmocom clone because it looks official, and you choose steve-m repo. I guess it's on purpose, do you have some information about this, or a recommendation?
- https://github.com/osmocom/rtl-sdr
- https://github.com/steve-m/librtlsdr

Reply:
I chose the `steve-m` one as it is listed as the official GitHub mirror at https://osmocom.org/projects/rtl-sdr/wiki/Rtl-sdr#Source-Code

Reply:
I will update my README file and use this repo.